niri-config/src/lib.rs

```diff
@@ -2281,6 +2295,10 @@ mod tests {
             debug {
                 render-drm-device "/dev/dri/renderD129"
             }
+
+            named-workspace "workspace-1"
+            named-workspace "workspace-2"
+            named-workspace "workspace-3"
```

Comment:
I was thinking, maybe just `workspace "name"`?

Reply:
I was thinking about that too, but then `niri msg action focus-workspace` vs `niri msg action focus-named-workspace` would have felt weird.

If it's possible to make it so that `focus-workspace` can work with either an `u8` or a `String`, that'd solve the problem. But I haven't tried that (yet).